domain-server/resources/web/web-new/src/css/app.scss

```diff
@@ -1,4 +1,34 @@
 // app global css in SCSS form
+:root {
```

Comment:
All of these global variables should be replaced with a link to the variables in the [Web Component Library](https://github.com/digisomni-syndicate/web-component-library/blob/10aae3f6eb7f5d0c3c2b86ce15bac40e4fa5383d/src/css/app.scss) in the near future. Please add a todo note here, or an issue for this so that we don't forget.